source/sphinx_extensions/command_block/extension.py

```diff
@@ -40,17 +63,34 @@ class CommandBlockDirective(docutils.parsers.rst.Directive):
     has_content = True
 
     option_spec = {
-        'no-exec': docutils.parsers.rst.directives.flag
+        'no-exec': docutils.parsers.rst.directives.flag,
+        'url': docutils.parsers.rst.directives.unchanged_required,
+        'rename': docutils.parsers.rst.directives.unchanged_required,
```

Comment:
👍 

Background on `rename` choice: was originally going to implement a "rename" using the HTML `a` tag attr `download`, which can change the name of the remote file. It didn't pan out, due to Cross-Origin issues.